set.mm

```diff
@@ -263599,6 +263599,8092 @@ the same dimension over the same (nonzero) ring.  (Contributed by AV,
     DOPUDUGSUEBBFCNQTRABUECUAUB $.
 
 
+$(
```

Comment:
The deleted part appears here without change.  At the end some hmldefs were also moved.  This reflects a different order of definitions now, I presume.